### PR TITLE
fix: function's returns are missing

### PR DESCRIPTION
**Pull request by summer-ji-eng**
Problem: client library's generated document miss functions' returns description.

Cause: `returns.tmpl` has never been referred before.

Fix: 
1. Add `returns.tmpl` in `method.tmpl`
2. Use `<table>`  like params and propers to render returns

Result:
image

### CODE REVIEW

File: static/styles/jsdoc-default.css [hunk 1]
```diff
@@ -1,569 +1,606 @@
 * {
```
Comment (alexander-fenster) [inline static/styles/jsdoc-default.css:1]: This file seems to have only formatting changes, can you please revert it to make it easier to review?  (or if there are changes other than formatting, they should be visible)

Comment (summer-ji-eng) [reply static/styles/jsdoc-default.css]: I added the format to change for class `returns` which I defined in `returns.tmpl`. Others are formatting changes, I reverted now.

Comment (alexander-fenster) [reply static/styles/jsdoc-default.css]: Thank you, the changes are much more visible now.

**Comment by summer-ji-eng** [comment]
Thanks for reviewing. 👍

**Comment by summer-ji-eng** [comment]
Looks like I don't have access to merge the pull requests.
@JustinBeckwith @bcoe could you help me merge it?

**Pull request merged**